src/components/FindMovie/FindMovie.tsx

```diff
@@ -28,30 +28,40 @@ export const FindMovie: React.FC<Props> = ({ setMovies }) => {
         imdbID,
       } = film;
 
-      if (film.Poster === 'N/A') {
-        setMovie({
-          title: Title,
-          description: Plot,
-          imgUrl: 'https://via.placeholder.com/360x270.png?text=no%20preview',
-          imdbUrl: `https://www.imdb.com/title/${imdbID}`,
-          imdbId: imdbID,
-        });
-      } else {
-        setMovie({
-          title: Title,
-          description: Plot,
-          imgUrl: Poster,
-          imdbUrl: `https://www.imdb.com/title/${imdbID}`,
-          imdbId: imdbID,
-        });
-      }
+      setMovie({
+        title: Title,
+        description: Plot,
+        imgUrl: (Poster !== 'N/A' ? Poster
+          : 'https://via.placeholder.com/360x270.png?text=no%20preview'),
+        imdbUrl: `https://www.imdb.com/title/${imdbID}`,
+        imdbId: imdbID,
+      });
     } else {
       setFailedRequest(true);
     }
 
     setLoading(false);
   };
 
+  const searchMovie = () => {
+    addMovie(query);
+    setLoading(true);
```

Comment:
why do you set loading `true` **after** fetching the movie? In `addMovie` you set it `false` in the end. So set it `true` at the beginning of `addMovie`

Also, I'd rename `addMovie` to smth like `fetchMovie`. This name will describe function better